pkg/proto/configuration/grpc/grpc.proto

```diff
@@ -35,6 +35,14 @@ message ClientConfiguration {
   //
   // Header names must be lower case.
   repeated string forward_metadata = 4;
+
+  // Map of gRPC metadata headers to set in client connection
+  // Header names must be lower case.
+  message header_values {
```

Comment:
Iirc best practice for message names is CamelCase and not snake_case?

Reply:
@mickael-carl it's protobuf file. See https://developers.google.com/protocol-buffers/docs/style

Reply:
`Use CamelCase (with an initial capital) for message names – for example, SongServerRequest. Use underscore_separated_names for field names (including oneof field and extension names) – for example, song_name. ` -> isn't that applicable for `header_values` ? (taken from [here](https://developers.google.com/protocol-buffers/docs/style#message-and-field-names))

Reply:
Exactly. In this case it's a message; not a field name.

Reply:
@mickael-carl you were right, fixed it